### PR TITLE
PLAT-2242: Zen-UI: Add disabled prop to zen-sidebar-nav-item

### PR DESCRIPTION
**Pull request by chavlji**
[JIRA ticket](https://reciprocitylabs.atlassian.net/browse/PLAT-2242)

### CODE REVIEW

File: src/components/zen-sidebar-nav-item/zen-sidebar-nav-item.tsx [hunk 1]
```diff
@@ -20,16 +20,16 @@ export class ZenSidebarNavItem {
...
+  @Prop({ reflect: true }) readonly disabled: boolean = false;
+
   @Watch('selected')
   selectedChanged(selected: boolean): void {
     if (!selected) {
       this.deselectAllChildren();
       return;
     }
-    const subitems = this.getItems();
-    if (subitems.length && !subitems.some(n => n.selected)) {
```
Comment (homer0) [inline src/components/zen-sidebar-nav-item/zen-sidebar-nav-item.tsx:30]: While this makes sense (I think the ticket for improvements mentions something related), how do we "open" an item?

Comment (chavlji) [reply src/components/zen-sidebar-nav-item/zen-sidebar-nav-item.tsx:30]: It still works fine. It just remove autoselect of the first child element (which is in our case Controls item which will be disabled):

https://user-images.githubusercontent.com/5729421/118478775-8e518900-b710-11eb-8889-a6bcb71d40c0.mov

I know it's still not perfect, but fixing it exactly as it should be, would take me much more time and I won't be ready till due date. We do have a [separate ticket to fix it as it should be](https://reciprocitylabs.atlassian.net/browse/PLAT-2103).

**Review by homer0 — APPROVED** [review]

File: src/components/zen-sidebar-nav-item/zen-sidebar-nav-item.tsx (hunk 1 above)
Comment (homer0) [reply src/components/zen-sidebar-nav-item/zen-sidebar-nav-item.tsx:30]: Sorry! I thought that it would only open if one of its children was selected.

**Pull request merged**